app/javascript/components/client-components/Nav/index.tsx

```diff
@@ -42,10 +42,11 @@ export const ClientNavLink = ({
   additionalPatterns?: string[];
   onClick?: (event: React.MouseEvent) => void;
 }) => {
-  const currentPath = window.location.href;
+  const currentPath = window.location.pathname + window.location.search;
 
   const ariaCurrent = [href, ...additionalPatterns].some((pattern) => {
-    const escaped = escapeRegExp(pattern);
+    const patternPath = pattern.includes("://") ? new URL(pattern).pathname + new URL(pattern).search : pattern;
+    const escaped = escapeRegExp(patternPath);
```

Comment:
Fixes nav active state by normalizing URLs to pathname + search before comparison. Previously full URL vs path comparison always failed.